tern/classes/image.py

```diff
@@ -16,6 +16,8 @@ class Image:
         manifest: the json object representing the image manifest
         config: the image config metadata
         layers: list of layer objects in the image
```

Comment:
We should probably update `layers` to say that it might not reflect the total number of layers as we only load the ones we need.